### DISCUSSION TITLE
MXM GPU on raspberry Pi 5

### DISCUSSION THREAD

**Discussion by Dieter-hamer** [geerlingguy/raspberry-pi-pcie-devices#603] [General]
Hi everyone. Hope all is well. 

I got an idea.. usually dangerous but not this time I don't think.. Wouldn't it be possible to mount a MXM GPU on a hat for the raspberry Pi 5? Something like a GT1030 or A350m arc graphics. I'm going to feed power in from the hat so we have enough power for everything

Any thoughts?

-Dietet

**Comment by RSC-Games** [comment]
Possible? Yes. Reasonable? Depends. A lot of companies were known to create their own MXM pinouts purely to make swapping cards more difficult. I was actually already thinking along the same lines for like an upgradeable "raspberry pi laptop" like the ARGON One but it exposes both PCIe0 and PCIe1 - PCIe0 to an NVMe and PCIe1 to an MXM module.
